Source/Android/app/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
```

Comment:
Did you test this change on Android 10? I previously also changed the SdkVersion to 29 but forgot that scoped storage is a thing, which might cause issues so I just went with 28 for now.

Reply:
targetSdkVersion 29 is what gives us scoped storage, right? I'm not aware of compileSdkVersion 29 doing the same.

Reply:
Ohh I think you're right. Should be good then!

Reply:
Maybe makes sense to explicitly set `targetSdkVersion 28` until we stress the scoped storage scenario better in Android Q devices?